shared/reducers/wallets.js

```diff
@@ -19,6 +19,47 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
       return state.setIn(['paymentsMap', action.payload.accountID], I.List(action.payload.payments))
     case WalletsGen.selectAccount:
       return state.set('selectedAccount', action.payload.accountID)
+    case WalletsGen.validateAccountName:
```

Comment:
to be super safe we should likely validate the params are still good, lets chat

Reply:
👍